### DISCUSSION TITLE
It doesn't seem to work properly in asynchronous server-side components.(nextjs13)

### DISCUSSION THREAD

**Discussion by crow-yang** [andreizanik/cookies-next#43] [Q&A] [answered]
It seems that the server component does not work normally.

For example

export default astnc function TestServerCompo{
   getCookies("something") <- no cookie but cookie is set on page!
}

is it right? should i use "import { cookies } from 'next/headers'"?

**Comment by andreizanik** [comment, marked as answer]
Hello! 
If you want to use it on the server side, call functions with `req` and `res`. 

[Example from the documentation](https://github.com/andreizanik/cookies-next#ssr-example)
